src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -564,6 +925,98 @@ public static void Test1(int x)
         }
 
         [Fact()]
+        public void AnonymousTypeSymbols_Error_Simple_OperationTree()
+        {
+            string source = @"
+class ClassA
+{
+    public static void Test1(int x)
+    /*<bind>*/{
+        object v1 = new
+        {
+            aa = xyz,
+            BB = """",
+            CCC = new SSS()
+        };
+
+        object v2 = new
+        {
+            aa = new SSS(),
+            BB = 123.456,
+            CCC = new
+            {
+                (new ClassA()).aa,
+                ClassA.BB,
+                ClassA.CCC
+            }
+        };
+    }/*</bind>*/
+}
+";
+            string expectedOperationTree = @"
+IBlockStatement (2 statements, 2 locals) (OperationKind.BlockStatement, IsInvalid) (Syntax: '{ ... }')
+  Locals: Local_1: System.Object v1
+    Local_2: System.Object v2
+  IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement, IsInvalid) (Syntax: 'object v1 = ... };')
```

Comment:
Same issue as above - IsInvalid needs to be changed.

---
In reply to: [123879697](https://github.com/dotnet/roslyn/pull/20364#discussion_r123879697) [](ancestors = 123879697)